cookbooks/aws-parallelcluster-slurm/recipes/install_slurm.rb

```diff
@@ -19,6 +19,30 @@
   action :purge
 end
 
+if node['platform'] == 'ubuntu'
+  apt_update
+
+  package 'libjson-c-dev' do
+    retries 3
+    retry_delay 5
+  end
+
+  package 'libhttp-parser-dev' do
+    retries 3
+    retry_delay 5
+  end
+else
+  package 'json-c-devel' do
+    retries 3
+    retry_delay 5
+  end
+
+  package 'http-parser-devel' do
+    retries 3
+    retry_delay 5
+  end
+end
+
```

Comment:
What about something like this? (you might need to double check the syntax:

```
slurm_build_deps = value_for_platformvalue_for_platform(
  'ubuntu' => {
    'default' => w%(libjson-c-dev libhttp-parser-dev),
  },
  'default' => w%(json-c-devel http-parser-devel),
)

package slurm_build_deps do
    retries 3
    retry_delay 5
end
```

Reply:
Good idea! Changed